test/deploy/kubernetes/template-config/cluster_redefined.yaml

```diff
@@ -2,6 +2,11 @@ cluster:
   name: cortx-cluster
   id: 0007ec45379e36d9fa089a3d615c32a3
   node_types:
+    - name: client_node        # Optional
+      components:
+    - name: cclient
+      services:
+      - cclient_service      # cclient_setup --services=cclient_service
```

Comment:
Add group key to the nodes section...to represent two pods running on the same node. 

```
cluster:
  nodes:
    - name: data-pod-1.1
      id: data-pod-id-1.1
      hostname: data-pod-1.1
      failure-group: data-node-1
    - name: data-pod-1.2
      id: data-pod-id-1.2
      hostname: data-pod-1.2
      failure-group: data-node-1
```

Reply:
Added **failure-group:** key for all data-nodes.